Mathlib/GroupTheory/Subgroup/Basic.lean

```diff
@@ -756,7 +756,7 @@ theorem coe_zpow (x : H) (n : ℤ) : ((x ^ n : H) : G) = (x : G) ^ n :=
 #align subgroup.coe_zpow Subgroup.coe_zpow
 #align add_subgroup.coe_zsmul AddSubgroup.coe_zsmul
 
-@[to_additive (attr := simp)]
+@[to_additive]
 theorem mk_eq_one_iff {g : G} {h} : (⟨g, h⟩ : H) = 1 ↔ g = 1 :=
```

Comment:
What was the reason for removing `simp` here?

Reply:
`AddSubmonoid.mk_eq_zero` proves it (as reported by simp nf)

Reply:
Can you add a comment to that effect? Also, for consistency, either the new lemma should be called `mk_eq_one_iff`, or you should rename the existing one.